src/sgame/sg_active.cpp

```diff
@@ -2186,8 +2186,7 @@ void ClientThink_real( gentity_t *self )
 
 		if ( ent && ent->use &&
 		     ( !ent->buildableTeam   || ent->buildableTeam   == client->pers.team ) &&
-		     ( !ent->conditions.team || ent->conditions.team == client->pers.team ) &&
-		     Distance( self->s.origin, ent->s.origin ) < ENTITY_USE_RANGE )
+		     ( !ent->conditions.team || ent->conditions.team == client->pers.team ) )
```

Comment:
This makes the use condition different on client than on server. This means that you can have some interesting bugs like this one where you *think* you can build things, but you actually can't. (notice the error message behind the dialog)

![image](https://user-images.githubusercontent.com/59283660/177044044-e8242566-580d-4905-9bf5-458de3b287f2.png)